cypress/support/index.ts

```diff
@@ -0,0 +1,7 @@
+Cypress.Commands.add("copyStorage", (fromStorage: { [name: string]: any }, toStorage: { [name: string]: any }) => {
```

Comment:
Codacy found an issue: ['Cypress' is not defined.](https://app.codacy.com/gh/Project-Lavinia/Lavinia-client/pullRequest?prid=5953345)